lib/shared/index.ts

```diff
@@ -27,17 +28,20 @@ export async function request<T>(
     );
   }
 
+  const finalHeaders = { ...fetchConfig.headers, ...requestConfig.headers };
+
   let response: Response;
   try {
     const body: BodyInit | undefined = requestConfig.data
       ? JSON.stringify(requestConfig.data)
       : requestConfig.dataRaw;
 
     response = await fetch(url.toString(), {
+      ...fetchConfig,
```

Comment:
`fetchConfig` needs to be last in the object construction in order to override the other properties set. Otherwise `fetchConfig.cache` gets clobbered here

Reply:
Hmm, I guess this should be
```
{
   method: requestConfig.method,
   body: body,
   cache: "no-store",
   ...fetchConfig,
   headers: finalHeaders,
}
```

Reply:
Ahh okay, makes sense